SharedSources/PresentationTheme.swift

```diff
@@ -0,0 +1,156 @@
+/*****************************************************************************
+ * VLC for iOS
+ *****************************************************************************
+ * Copyright (c) 2018 VideoLAN. All rights reserved.
+ * $Id$
+ *
+ * Authors: Carola Nitz <caro # videolan.org>
+ *
+ * Refer to the COPYING file of the official project for license.
+ *****************************************************************************/
+
+import Foundation
+
+extension Notification.Name {
+    static let VLCThemeDidChangeNotification = Notification.Name("themeDidChangeNotfication")
+}
+
+public class ColorPalette : NSObject {
```

Comment:
I think we could use the modifier @objcMembers to ColorPalette, to avoid individually exposing to objc all its the properties.

Reply:
Ohhhh didn't know about that one yet <3 awesome today I learned